_layouts/homepage.html

```diff
@@ -70,8 +70,16 @@ <h1>{{ page.secondary.heading }}</h1>
       </a>
       {% endif %}
       <a href="{{ page.ctas.roadmap.url }}" class="link-readmore" target="_blank">{{ page.ctas.roadmap.text }}</a>
-      {% assign latest = site.versions | sort: 'version_sort' | reverse | first %}
-      <div class="latest-version">{{ page.version_feature.latest_label}} {{ latest.version }} {{ page.version_feature.date_label}} {{ latest.date | date_to_string: "ordinal", "US"}}</div>
+      {% assign versions = site.versions | sort: 'version_sort' | reverse %}
+      {% assign latest = versions | first %}
```

Comment:
1. First lets add a section below this line. This will calculate what the previous major should be: 

```{% assign previous_major = latest.version | split: "." | first | plus: -1 %}```

Note, the plus here converts the version string to an int which we have to account for later.